cloud-mock/src/main/java/com/sequenceiq/cloudbreak/cloud/mock/MockResourceConnector.java

```diff
@@ -193,17 +193,17 @@ public String getDBStackTemplate() throws TemplatingDoesNotSupportedException {
     }
 
     @Override
-    public void startDatabaseServer(AuthenticatedContext authenticatedContext, String dbInstanceIdentifier) {
+    public void startDatabaseServer(AuthenticatedContext authenticatedContext, DatabaseStack stack) {
         throw new UnsupportedOperationException("Database server start operation is not supported for " + getClass().getName());
     }
 
     @Override
-    public void stopDatabaseServer(AuthenticatedContext authenticatedContext, String dbInstanceIdentifier) {
+    public void stopDatabaseServer(AuthenticatedContext authenticatedContext, DatabaseStack stack) {
         throw new UnsupportedOperationException("Database server stop operation is not supported for " + getClass().getName());
     }
 
     @Override
-    public ExternalDatabaseStatus getDatabaseServerStatus(AuthenticatedContext authenticatedContext, String dbInstanceIdentifier) throws Exception {
+    public ExternalDatabaseStatus getDatabaseServerStatus(AuthenticatedContext authenticatedContext, DatabaseStack stack) throws Exception {
         throw new UnsupportedOperationException("Database server status lookup is not supported for " + getClass().getName());
```

Comment:
Question: during a mock test is the status sync running? If yes, it might then break the test.

Reply:
Do you mean the exception might break the test? I don't think it is the case as I'm not aware of any breaking test in connection with this exception. It got introduced more than a month ago.